exporter.py

```
@@ -376,13 +378,44 @@ def anki_request(action, **params):
+            import re
+            try:
+                tokens = shlex.split(search_query)
+            except ValueError:
+                tokens = search_query.split()
+
+            clean_tokens = []
+            for token in tokens:
+                if token.startswith("-"):
+                    continue
+                # Strip leading/trailing quotes (helps with malformed shlex fallback)
+                token = token.strip('"')
```

Ruff (0.16.3)

[warning] `for` loop variable `token` overwritten by assignment target

(PLW2901)